job/job_test.go

```diff
@@ -34,12 +34,14 @@ func TestUnmarshalJSON(t *testing.T) {
 		`{"aggr_id":"timeoutfoo", "download_timeout":-2, "url":"http://foobar.com","callback_url":"http://foo.bar","extra":"whatever"}`:   true,
 		`{"aggr_id":"timeoutfoo", "download_timeout":"4", "url":"http://foobar.com","callback_url":"http://foo.bar","extra":"whatever"}`:  true,
 
-		// user agent
-		`{"aggr_id":"useragentfoo", "user_agent":"Downloader Test", "url":"http://foobar.com","callback_url":"http://foo.bar","extra":"whatever"}`: false,
-		`{"aggr_id":"useragentfoo", "url":"http://foobar.com","callback_url":"http://foo.bar","extra":"whatever"}`:                                 false,
-		`{"aggr_id":"useragentfoo", "user_agent":"", "url":"http://foobar.com","callback_url":"http://foo.bar","extra":"whatever"}`:                false,
-		`{"aggr_id":"useragentfoo", "user_agent":null, "url":"http://foobar.com","callback_url":"http://foo.bar","extra":"whatever"}`:              true,
-		`{"aggr_id":"useragentfoo", "user_agent":3, "url":"http://foobar.com","callback_url":"http://foo.bar","extra":"whatever"}`:                 true,
+		`{"aggr_id":"requestheadersfoo", "url":"http:foobar.com", "callback_url":"http://foo.bar",
```

Comment:
Keep the url field in these testcases as it was... 
(missing `//` )

Reply:
Ah! Yes this was probably messed-up during a rebase.. sorry.. fixed!